velox/common/caching/ScanTracker.h

```diff
@@ -161,8 +157,7 @@ class ScanTracker {
   const std::function<void(ScanTracker*)> unregisterer_{nullptr};
   FileGroupStats* const fileGroupStats_;
 
-  std::mutex mutex_;
-  folly::F14FastMap<TrackingId, TrackingData> data_;
+  folly::Synchronized<folly::F14FastMap<TrackingId, TrackingData>> data_;
```

Comment:
```
folly::Synchronized<folly::F14FastMap<TrackingId, std::shared_ptr<TrackingData>>> data_;
```

Reply:
Yeah this is a good catch, let's use `unique_ptr` here